### ISSUE TITLE
Cannot install npm package

### ISSUE THREAD

**Issue by dsajanice** [Azure-Samples/azure-iot-samples-node#28]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ x] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
>

Following the steps in https://docs.microsoft.com/en-us/azure/iot-hub/quickstart-device-streams-echo-nodejs#communicate-between-device-and-service-via-device-streams to start proxy.js. I am hitting the following error when trying to install the npm package.

jadsa-auto@jadsa-auto-control:~/Repos/azure-iot-samples-node/iot-hub/Quickstarts/device-streams-service$ npm install azure-iothub@streams-preview
device-streams-service-samples@0.0.1 /home/jadsa-auto/Repos/azure-iot-samples-node/iot-hub/Quickstarts/device-streams-service
└── azure-iothub@1.10.0-preview.0  invalid

jadsa-auto@jadsa-auto-control:~/Repos/azure-iot-samples-node/iot-hub/Quickstarts/device-streams-service$ npm install
npm ERR! Linux 4.15.0-1036-azure
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install"
npm ERR! node v8.10.0
npm ERR! npm  v3.5.2

npm ERR! No compatible version found: azure-iothub@1.10.0-preview
npm ERR! Valid install targets:
npm ERR! 2.0.0-dtpreview, 1.10.0-preview.0, 1.9.4, 1.9.3, 1.9.2, 1.9.1, 1.9.0, 1.9.0-preview.0, 1.8.3, 1.8.2, 1.8.1, 1.8.0, 1.7.5, 1.7.4, 1.7.3, 1.7.1, 1.7.0, 1.7.0-modules-preview.2, 1.7.0-modules-preview.1, 1.4.1-modules-preview.2, 1.4.0-modules-preview.2, 1.3.2, 1.3.1, 1.3.0, 1.3.0-modules-preview, 1.2.5, 1.2.4, 1.2.3, 1.2.2, 1.2.1, 1.2.0, 1.1.18, 1.1.17, 1.1.16, 1.1.15, 1.1.14, 1.1.13, 1.1.12, 1.1.11, 1.1.10, 1.1.9, 1.1.8, 1.1.7, 1.1.6, 1.1.5, 1.1.4, 1.1.3, 1.1.2, 1.1.1, 1.1.0, 1.1.0-dtpreview.2, 1.1.0-dmpreview.4, 1.1.0-dmpreview.3, 1.1.0-dmpreview.2, 1.1.0-dmpreview.1, 1.1.0-dmpreview.0, 1.0.19, 1.0.18, 1.0.17, 1.0.16, 1.0.15, 1.0.14, 1.0.13, 1.0.12, 1.0.11, 1.0.10, 1.0.9, 1.0.8, 1.0.7, 1.0.6, 1.0.5, 1.0.4, 1.0.3, 1.0.2, 1.0.1, 1.0.0, 1.0.0-preview.11, 1.0.0-preview.10, 1.0.0-preview.9, 1.0.0-preview.8, 1.0.0-preview.7, 1.0.0-preview.6, 1.0.0-preview.5, 1.0.0-preview.4, 1.0.0-preview.3, 0.0.1, 0.0.1-preview.3
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /home/jadsa-auto/Repos/azure-iot-samples-node/iot-hub/Quickstarts/device-streams-service/npm-debug.log
jadsa-auto@jadsa-auto-control:~/Repos/azure-iot-samples-node/iot-hub/Quickstarts/device-streams-service$

### Any log messages given by the failure
>

### Expected/desired behavior
>

### OS and Version?
>  Linux 4.15.0-1036-azure

### Versions
>

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by pierreca** [comment]
Fix: #29

the second command (`npm install`) failed because the dependency version in package.json was incorrect.
The first command should have succeeded given 1.10.0-preview.0 shows as a valid target (also: just tested, it worked, so I wonder if it was a temporary npm hiccup).

**Comment by pierreca** [comment]
closing since the fixed was checked in. @dsajanice if you find out what went wrong on the first command, let me know.
